app/helpers/application_helper/button/host_feature_button.rb

```diff
@@ -1,13 +1,15 @@
 class ApplicationHelper::Button::HostFeatureButton < ApplicationHelper::Button::GenericFeatureButton
 
   def visible?
+    # TODO: @feature.nil? || @record.nil? || @record.supports?(@feature)
+    # TODO: ensure start stop are supported in openstack
     unless @feature.nil? || @record.nil?
       if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager)
         return true if %w[start stop].include?(@feature.to_s)
         return false
       end
 
-      return @record.is_available?(@feature) if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager::Host)
+      return @record.supports?(@feature) if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager::Host)
```

Comment:
Unrelated to this change, but really want to get rid of this I'm very confused why this is in a seemingly generic "host_feature_button"